javers-core/src/main/java/org/javers/repository/jql/QueryBuilder.java

```diff
@@ -509,6 +509,17 @@ public QueryBuilder withCommitPropertyLike(String name, String value){
         return this;
     }
 
+    /**
+     * Only snapshots with a given commit property partially containing a given value, ignoring case.<br/>
+     * Equivalent to SQL clause: WHERE lower(property_value) LIKE lower('%value%')
+     *
+     */
+    public QueryBuilder withCommitPropertyLikeIgnoreCase(String name, String value) {
```

Comment:
the javadoc for `withCommitPropertyLike()` says pretty much the same. I suggest to force the existing method to be case insensitive

Reply:
Do you mean I should change the existing function to also ignore case when searching, rather than creating a new separate method? :)

Reply:
exactly